activestorage/test/models/blob_test.rb

```diff
@@ -41,6 +41,14 @@ class ActiveStorage::BlobTest < ActiveSupport::TestCase
     end
   end
 
+  test "purge removes from external service" do
+    blob = create_blob
+    blob_key = blob.key
```

Comment:
This local variable shouldn't be necessary, since `blob` still holds a reference to the blob after it’s purged:

```ruby
blob.purge
assert_not ActiveStorage::Blob.service.exist?(blob.key)
```